webapp/src/pages/createApplicationForm/Form.css

```diff
@@ -423,3 +481,66 @@
   background-color: gray;
   color: white;
 }
+
+html {
```

Comment:
I am sorry I am not sure I follow on this one. If you could break it down for me plz!

Reply:
Hi @Malaba6 - I just mean you're setting the style for the entire html element in the createApplicationForm style sheet, which is problematic for two reasons:
1. We already set the font [here](https://github.com/deep-learning-indaba/Baobab/blob/a5b4fa192dfe86869b7df01bcb5cbb8914b69b24/webapp/src/App.css#L30) so I don't understand why this style is necessary
2. The createApplicationForm style sheet should only contain styles that apply to the createApplicationForm component/page and not the entire site

Please also prefix your other styles like .circle etc with the .application-form-wrap to prevent name-clashes with other components that might use these common names.

Reply:
> Hi @Malaba6 - I just mean you're setting the style for the entire html element in the createApplicationForm style sheet, which is problematic for two reasons:
> 
> 1. We already set the font [here](https://github.com/deep-learning-indaba/Baobab/blob/a5b4fa192dfe86869b7df01bcb5cbb8914b69b24/webapp/src/App.css#L30) so I don't understand why this style is necessary
> 2. The createApplicationForm style sheet should only contain styles that apply to the createApplicationForm component/page and not the entire site
> 
> Please also prefix your other styles like .circle etc with the .application-form-wrap to prevent name-clashes with other components that might use these common names.

Oh my bad! those styles were just meant for some experiment, I removed them. Thanx!